src/components/NotesButton.tsx

```diff
@@ -9,34 +9,34 @@ export default function NotesButton(): JSX.Element {
         <TouchableOpacity onPress = {() => setModalVisibility(true)}>
             <Text style = {styles.buttonText}>+ Notes </Text>    
             <Modal isVisible = {isModalVisible}>
-                <KeyboardAvoidingView style = {{flex: 1}} behavior = "padding">
-                    <View style = {styles.modalContainer} > 
-                            <Text style = {styles.title}>Current Note</Text>  
+                <View style = {styles.modalContainer} > 
+                        <Text style = {styles.title}>Notes</Text>  
+                        <View style={{borderWidth: .75, flex: 1}}>
                             <ScrollView>                      
                                 <TextInput 
                                     style = {styles.noteInput}
```

Comment:
This is a common occurrence throughout this file. Can you go ahead and remove the spaces around the `=` sign just for the React components? I know that we didn't make a standard for this, but it will look consistent with the other files in our code.

```suggestion
                                    style={styles.noteInput}
```